src/cf-forms/usage.md

```diff
@@ -22,39 +22,44 @@ Capital Framework.
     - [Label helper text](#label-helper-text)
 - [Inputs](#inputs)
     - [Basic Text Inputs](#basic-text-inputs)
-        - [Default state](#default-state)
-        - [Hover state](#hover-state)
-        - [Focus state](#focus-state)
-        - [Disabled state](#disabled-state)
+        - [Text input default state](#text-input-default-state)
+        - [Text input hover state](#text-input-hover-state)
+        - [Text input focus state](#text-input-focus-state)
+        - [Text input disabled state](#text-input-disabled-state)
         - [Full-width modifier](#full-width-modifier)
     - [Basic checkboxes](#basic-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Checkbox default state](#checkbox-default-state)
+        - [Checkbox hover state](#checkbox-hover-state)
+        - [Checkbox focus state](#checkbox-focus-state)
+        - [Checkbox selected state](#checkbox-selected-state)
+        - [Checkbox disabled state](#checkbox-disabled-state)
     - [Basic radio buttons](#basic-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Radio button default state](#radio-button-default-state)
```

Comment:
Same comment here as above for checkboxes.

Reply:
See comment above about links needing a unique name.